docs/pages/router/basics/navigation.mdx

```diff
@@ -207,6 +207,29 @@ export default function Page() {
 }
 ```
 
+## Prefetching
+
+The `prefetch` prop on a `<Link />` component enables prefetching of the target screen when the component is rendered. This allows for faster navigation by preparing the screen in advance.
+
+```tsx
+import { Redirect } from 'expo-router';
+
+export default function Page() {
+  return <Link href="/about" prefetch />;
+}
+```
+
+When `prefetch` is set, Expo Router will attempt to render the target screen off-screen. The exact behavior depends on the type of navigator used:
+
+- **Expo Router Navigators**: Render the target screen off-screen to enable preloading.
+- **Custom Navigators**: May implement prefetching differently or not support it at all.
+
+When a screen is preloaded in a stack navigator, it will have a few limitations:
+
+- It cannot use the imperative `router` API.
```

Comment:
it may be worth clarifying/linking (from a technical standpoint) as to why this is not possible in this case

Reply:
Unfortunately this is a limitation of React Navigation that Expo Router inherits. I've included a link to the relevant React Navigation docs